src/Symfony/Bridge/PhpUnit/ErrorAssert.php

```diff
@@ -46,7 +46,7 @@ public static function assertErrorsAreTriggered($expectedType, $expectedMessages
         $triggeredMessages = array();
 
         try {
-            $prevHandler = set_error_handler(function ($type, $message, $file, $line, $context) use ($expectedType, &$triggeredMessages, &$prevHandler) {
```

Comment:
this change is broken. The variable is passed by reference to the closure, so changing the reference is not an unused variable.